.github/workflows/acceptance.yml

```diff
@@ -19,6 +19,9 @@ jobs:
     runs-on: ubuntu-20.04
     timeout-minutes: 20
     strategy:
+      # This helps not having to run multiple jobs because one fails, thus, reducing resource usage
+      # and reducing the risk that one of many runs would turn red again (read: intermittent tests)
+      fail-fast: false
```

Comment:
@billyvg I tried using `defaults` at the workflow level, however, that also requires defining `matrix` at the workflow level, thus, going back to this.